src/prefect/utilities/filesystem.py

```diff
@@ -19,10 +19,11 @@ def create_default_ignore_file(path: str) -> bool:
     whether a file was created.
     """
     path = pathlib.Path(path)
-    if (path / ".prefectignore").exists():
+    ignore_file = path / ".prefectignore"
+    if ignore_file.exists():
         return False
-    default_file = pathlib.Path(__file__).parent / ".." / ".prefectignore"
-    with open(path / ".prefectignore", "w") as f:
+    default_file = pathlib.Path(__file__).parent.parent / ".prefectignore"
```

Comment:
I'm not sure if this is equivalent, flagging this change

Reply:
This should use `prefect.__module_path__` and specify the path relative to the `prefect` module the instead of the current file (`__file__.parent ...`)

Reply:
So like `prefect.__module_path__ / ".prefectignore"`?

The default one seems to be in the module root: https://github.com/PrefectHQ/prefect/blob/main/src/prefect/.prefectignore

Reply:
Updated and tested manually using:

```
>>> import prefect.utilities.filesystem as fs

>>> fs.create_default_ignore_file("/tmp")
True
```

I checked and the file exists under `/tmp/.prefectignore`